zubhub_frontend/zubhub/src/views/profile/DisplayBadges.js

```diff
@@ -0,0 +1,71 @@
+export function allBadgesTitle(projects) {
+    const {comments, likes, views}= addValues(projects);
+    let badges= [];
+
+    const title_for_projects=projectBadgeTitle(projects.length);

```

Comment:
Its better to stick to single case either camelCase or snake_case instead of mixing up

Reply:
In all the JS projects i have worked or seen its all camel case, But its completely up to the team. I am sure if there are any

@tuxology

Reply:
okay, I'll stick to the camelCase only.